docs/docs/legal/support-services.md

```diff
@@ -0,0 +1,106 @@
+
+Excluded are broader consulting & customer-specific engineering requests regarding use of ZITADEL Cloud. Moreover first level support requests by Customer’s end users shall be handled by Customer directly.
+
+### Support service process
+
+The customer may submit support requests (“ticket”) through any means of eligible communication channels, consisting of
+- Single discrete problem, issue, or request
+- Initial severity level and impact statement for assessment
+- Description of the issue and if possible a description of the observed and expected behavior,  steps to reproduce the issue, evidence that issue is not caused by connectivity / compute, relevant anonymized log-files etc.
+- All information requested by CAOS as we resolve the ticket (e.g. system logs)
+
+CAOS  will review the case information and determine the severity level (see below), working with the customer to assess the urgency of the request and use reasonable efforts to respond to support requests within the initial response time 
+
+CAOS will use reasonable efforts to resolve support request as defined below, but does not guarantee a workaround, resolution or resolution time
+
+Severity Level | Description 
+--- | --- 
+**Severity 1**<br/> Critical / Service down| <p>Widespread failure or complete unavailability of CAOS Core Services. </p><p> CAOS will use continuous effort to provide a workaround or permanent solution. When Core Services are available, the severity will be lowered to the new appropriate level.</p>
+**Severity 2**<br/> Core functionality unavailable or severely degraded| <p>WCore Services of CAOS software continue to operate in severely restricted fashion, yet long-term productivity may be impacted.</p><p> When Core Services are no longer severely degraded (eg, through a viable workaround or release), the severity level will be lowered to Severity 3.</p>
+**Severity 3**<br/>Standard support request| <p>Partial and non-critical loss of CAOS software functionality or major software defect, yet a workaround exists for viable long-term operation.</p><p>CAOS will continue to work on developing permanent resolution.</p>
+**Severity 4**<br/>Non-urgent request| <p>Defined as follows: <ul><li>Request for information or general query</li><li>Feature request</li><li>Performance issues and little to none functional impact</li><li>Defects with workarounds and little to low functional impact</li></ul></p>
+<p>CAOS will continue to work on developing permanent resolution and response to general requests. CAOS does not provide a timeline or guarantee to include any feature requests.</p>
+
+### Escalation
+
+The customer may escalate support requests following the escalation process: 
+1. For non-urgent needs, the client may request management escalation within the ticket. A manager will review the request and provide a response within one business day.
+2. For urgent needs, the client may escalate directly by calling +41 43 456 84 69 and emailing to hi@caos.ch
```

Comment:
were does this phone point to?

Reply:
This goes to my number. I think this is ok for now.

Reply:
ok